spec/factories/file_type.rb

```diff
@@ -124,6 +124,16 @@
     singleton_file_type
   end
 
+  factory :dot_pylintrc, class: 'FileType' do
+    created_by_admin
+    editor_mode { 'ace/mode/plain_text' }
+    executable
+    file_extension { '.pylintrc' }
+    indent_size { 4 }
+    name { 'Python' }
```

Comment:
Since we are not creating a `.py` file, I would consider another name more appropriate.

Reply:
fixed